src/item_pocket.cpp

```diff
@@ -0,0 +1,1051 @@
+                           float insulation, const temperature_flag flag )
+{
+    bool processed = false;
+    for( auto it = contents.begin(); it != contents.end(); ) {
+        if( _sealed ) {
+            // Simulate that the item has already "rotten" up to last_rot_check, but as item::rot
+            // is not changed, the item is still fresh.
+            it->set_last_rot_check( calendar::turn );
+        }
+        if( it->process( carrier, pos, activate, type.insulation_factor * insulation, flag ) ) {
+            it = contents.erase( it );
+            processed = true;
+        } else {
+            ++it;
+        }
+    }
+    return processed;
+}
+
+void item_pocket::remove_all_ammo( Character &guy )
+{
+    for( auto iter = contents.begin(); iter != contents.end(); ) {
+        if( iter->is_irremovable() ) {
+            iter++;
+            continue;
+        }
+        drop_or_handle( *iter, guy );
+        iter = contents.erase( iter );
+    }
+}
+
+void item_pocket::remove_all_mods( Character &guy )
+{
+    for( auto iter = contents.begin(); iter != contents.end(); ) {
+        if( iter->is_toolmod() ) {
+            guy.i_add_or_drop( *iter );
+            iter = contents.erase( iter );
+        } else {
+            ++iter;
+        }
+    }
+}
+
+void item_pocket::set_item_defaults()
+{
+    for( item &contained_item : contents ) {
+        /* for guns and other items defined to have a magazine but don't use "ammo" */
+        if( contained_item.is_magazine() ) {
+            contained_item.ammo_set(
+                contained_item.ammo_default(), contained_item.ammo_capacity() / 2
+            );
+        } else { //Contents are batteries or food
+            contained_item.charges =
+                item::find_type( contained_item.typeId() )->charges_default();
+        }
+    }
+}
+
+static void insert_separation_line( std::vector<iteminfo> &info )
+{
+    if( info.empty() || info.back().sName != "--" ) {
+        info.push_back( iteminfo( "DESCRIPTION", "--" ) );
+    }
+}
+
+void item_pocket::general_info( std::vector<iteminfo> &info, int pocket_number,
+                                bool disp_pocket_number ) const
+{
+    const std::string space = "  ";
+
+    if( disp_pocket_number ) {
+        const std::string pocket_translate = _( "Pocket" );
+        const std::string pocket_num = string_format( "%s %d:", pocket_translate, pocket_number );
+        info.emplace_back( "DESCRIPTION", pocket_num );
+    }
+    if( data->rigid ) {
+        info.emplace_back( "DESCRIPTION", _( "This pocket is <info>rigid</info>." ) );
+    }
+    if( data->min_item_volume > 0_ml ) {
+        const std::string min_vol_translation = _( "Minimum volume of item allowed:" );
+        info.emplace_back( "DESCRIPTION", string_format( "%s <neutral>%s</neutral>", min_vol_translation,
+                           vol_to_string( data->min_item_volume ) ) );
```

Comment:
```suggestion
        info.emplace_back( "DESCRIPTION", string_format( _( "Minimum volume of item allowed: <neutral>%s</neutral>" ),
                           vol_to_string( data->min_item_volume ) ) );
```
And the same for `max_vol_translation`, `max_weight_translation`, and `base_moves_translation` below.